benchopt/benchmark.py

```diff
@@ -549,14 +549,14 @@ def _extract_parameters(string):
     all_matches += re.findall(r'"[^\'"]*"', string)
     for match in all_matches:
         string = string.replace(match, str(hash(match)))
+    # print("###############", string)
 
     # Second, add quotes to all variable names (foo -> 'foo').
-    string = re.sub(r"[a-zA-Z][a-zA-Z0-9_-]*", r"'\g<0>'", string)
+    string = re.sub(r"[a-zA-Z][a-zA-Z0-9._-]*", r"'\g<0>'", string)
```

Comment:
Don't you need to escape the dot in a regex?

Reply:
In a character class the dot doesn't need to be escaped. But I'll add the tests as @TomDLT suggested

Reply:
Ok thanks